packages/mds-metrics-sheet/metrics-log.ts

```diff
@@ -78,6 +79,23 @@ async function appendSheet(sheetName: string, rows: MetricsSheetRow[]) {
   }
 }
 
+function eventCountsToStatusCounts(events: { [s in VEHICLE_EVENT]: number }) {
+  return Object.keys(events).reduce(
+    (acc: { [s in VEHICLE_STATUS]: number }, event) => {
+      return Object.assign(acc, { [EVENT_STATUS_MAP[event as VEHICLE_EVENT]]: event })
```

Comment:
Should't this be an addition instead of assignment since multiple event types map to a single status?

Reply:
Yep! Good catch 🤟🏼